malaria24/ona/tests/test_tasks.py

```diff
@@ -20,6 +21,10 @@ def setUp(self):
                       status=200, content_type='application/json',
                       body=pkg_resources.resource_string(
                           'malaria24', 'ona/fixtures/responses/data.json'))
+        post_save.disconnect(alert_new_case, sender=ReportedCase)
+
+    def tearDown(self):
+        post_save.connect(alert_new_case, sender=ReportedCase)
```

Comment:
I'm disconnecting the signal handler here for the tests runs because the SMS sending is not something I'm wanting to test here.